src/istio/control/client_context_base.cc

```diff
@@ -98,17 +99,20 @@ CancelFunc ClientContextBase::SendCheck(TransportCheckFunc transport,
     on_done(check_response_info);
   };
 
-  // TODO: add debug message
-  // GOOGLE_LOG(INFO) << "Check attributes: " <<
-  // request->attributes->DebugString();
+  if (MIXER_DEBUG_ENABLED) {
```

Comment:
Look each MIXER_DEBUG has this "if MIXER_DEBUG_ENABLE" check.   Can they be combined?

Reply:
Wonderful idea if I can make it work.  I'll do it if I can ensure that log arguments won't be serialized into nice pretty printed messages, etc when the log threshold wouldn't log them.

Reply:
Done. src/istio/utils/logger_test.cc shows that arguments to the logging macros will not be needlessly invoked if the log message is under the logging threshold.